server/system/service/attachment.go

```diff
@@ -198,8 +200,49 @@ func (svc attachment) CreateApplicationAttachment(ctx context.Context, name stri
 	return att, svc.recordAction(ctx, aaProps, AttachmentActionCreate, err)
 }
 
+func (svc attachment) CreateAuthAttachment(ctx context.Context, name string, size int64, fh io.ReadSeeker, labels map[string]string) (att *types.Attachment, err error) {
+	var (
+		aaProps       = &attachmentActionProps{}
+		currentUserID = intAuth.GetIdentityFromContext(ctx).Identity()
+	)
+
+	// check the file type
+	ext := filepath.Ext(name)
+	if ext != ".jpg" && ext != ".jpeg" && ext != ".png" {
```

Comment:
Should we allow gifs as well?

Reply:
I think we shouldn't allow gifs as profile avatars since they could be distracting and in cases where Corteza is used as a professional tool, I think it may be inappropriate.